libcxx/utils/libcxx/test/features.py

```diff
@@ -500,6 +501,43 @@ def _getAndroidDeviceApi(cfg):
     ),
 ]
 
+
+# Detect whether dbx is on the system.
+def check_dbx(cfg):
+    dbx_path = shutil.which("dbx")
+    if dbx_path is None:
+        return False
+
+    return True
+
+
+DEFAULT_FEATURES += [
+    Feature(
+        name="host-has-dbx",
+        when=check_dbx,
+        actions=[AddSubstitution("%{dbx}", lambda cfg: shutil.which("dbx"))],
+    )
+]
+
+
+# Detect whether LLDB is on the system.
+def check_lldb(cfg):
```

Comment:
This does not detect Python support in LLDB the name of the associated feature implies support `host-has-lldb-with-python`. I assume the tests require Python support so this looks like it needs more work. Maybe look at the LLDB test suite how they determine Python support.